examples/dbt-on-cloud-composer/terraform/main.tf

```diff
@@ -0,0 +1,179 @@
+module "composer_project" {
+  source          = "github.com/terraform-google-modules/cloud-foundation-fabric//modules/project?ref=v8.0.0"
+  parent          = var.root_node
+  billing_account = var.billing_account_id
+  prefix          = var.prefix
+  name            = "<gcp_project_name>"
+  iam_additive = {
+    "roles/owner" = var.owners
+  }
+  # Required for Cloud Composer
+  policy_boolean = {
+    "constraints/compute.requireOsLogin" = false
+  }
+  services        = [
+    "composer.googleapis.com",
+    "sourcerepo.googleapis.com",
+    "bigquery.googleapis.com",
+    "iamcredentials.googleapis.com"
+  ]
+}
+
+module "composer_vpc" {
+  source     = "github.com/terraform-google-modules/cloud-foundation-fabric//modules/net-vpc?ref=v8.0.0"
+  project_id = module.composer_project.project_id
+  name       = "composer-vpc"
+  subnets = [{
+    ip_cidr_range      = "10.1.0.0/24"
+    name               = "default"
+    region             = var.region
+    secondary_ip_range = {
+      pods     = "10.16.0.0/14"
+      services = "10.20.0.0/24"
+    }
+  }]
+  subnet_private_access = {
+    "subnet" = true
+  }
+}
+
+module "dbt_bucket" {
+  source        = "github.com/terraform-google-modules/cloud-foundation-fabric//modules/gcs?ref=v8.0.0"
```

Comment:
I don't think this is the correct repository, the correct one is: `https://github.com/GoogleCloudPlatform/cloud-foundation-fabric`. Maybe also update to the latest release of Fabric?